src/chrome/content/rules/AMetSoc.org.xml

```diff
@@ -41,7 +41,11 @@
 <ruleset name="AMetSoc.org (partial)">
 
 	<target host="ametsoc.org" />
-	<target host="*.ametsoc.org" />
+	<target host="annual.ametsoc.org" />
+	<target host="bookstore.ametsoc.org" />
+	<target host="journals.ametsoc.org" />
+	<target host="shib.ametsoc.org" />
+	<target host="www.ametsoc.org" />
 		<!--
```

Comment:
This is not a big deal but the [rulesets style guidelines](https://github.com/EFForg/https-everywhere/blob/master/CONTRIBUTING.md#target-ordering) recommend that `www.example.org` is listed right after `example.org`. It would be great if you have an easy way of  fixing that!

Reply:
Hmm I think I can do that, but that requires some extra effort to achieve exactly same sorting as in the example of style guidelines since pure `.sort()` would give rather:

* a.example.com
* a.example.net
* b.a.example.com
* b.example.com
* example.com
* example.net
* www.a.example.com
* www.example.com
* www.example.net

Reply:
Okay, I will let you and @Hainish decide if that's worth the effort then. I just thought I would point it out.

Reply:
Btw why `www.example.com` comes earlier than `a.example.com` in that example if it's supposed to be alphabetical?

Reply:
I can do something like this if it works for you:

* example.com
* a.example.com
* b.a.example.com
* www.a.example.com
* b.example.com
* www.example.com
* example.net
* a.example.net
* www.example.net

Reply:
So for this file it would become:

```
	<target host="ametsoc.org" />
	<target host="shib.ametsoc.org" />
	<target host="bookstore.ametsoc.org" />
	<target host="annual.ametsoc.org" />
	<target host="journals.ametsoc.org" />
	<target host="www.ametsoc.org" />
```

Does this look better?

Reply:
Alternatively, this:

```
	<target host="ametsoc.org" />
	<target host="annual.ametsoc.org" />
	<target host="bookstore.ametsoc.org" />
	<target host="journals.ametsoc.org" />
	<target host="shib.ametsoc.org" />
	<target host="www.ametsoc.org" />
```

Reply:
@Bisaloo thoughts on which one is better?

Reply:
But then things like this get messy...

```
	<target host="gateway10.whoson.com" />
	<target host="gateway11.whoson.com" />
	<target host="gateway3.whoson.com" />
	<target host="gateway4.whoson.com" />
	<target host="gateway5.whoson.com" />
	<target host="gateway6.whoson.com" />
	<target host="gateway7.whoson.com" />
	<target host="gateway8.whoson.com" />
	<target host="gateway9.whoson.com" />
```

Reply:
Uhm, no, you previous message looked fine but it's only because all example subdomains were only 1 letter long. Are you proposing to sort by reversing the target string?

If so, I think it's actually worse than the current commit.

Reply:
Oh, I didn't notice you commented again in the meantime. I just put emojis on your comments for a simpler way to express my opinion.

Reply:
> Alternatively, this:

	<target host="ametsoc.org" />
	<target host="annual.ametsoc.org" />
	<target host="bookstore.ametsoc.org" />
	<target host="journals.ametsoc.org" />
	<target host="shib.ametsoc.org" />
	<target host="www.ametsoc.org" />

What's the difference with the current commit?

Reply:
Ha, you're right, there is none. Just tried to actually implement sorting as per guidelines, but `www` seems to be somehow special-cased despite the guideline saying alphabetic order.

Reply:
I guess I'll just leave it as-is for now, but if you have a formalized idea for sorting, please let me know and I'll be happy to implement. Heading home for now, it's pretty late here.

Reply:
Okay, no worries, I was just nitpicking. We can sort this out next time the rulesets are manually edited.

Reply:
@RReverser It's only a nicety, for the purposes of this commit I don't think we should strictly enforce it.